game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts

```diff
@@ -99,27 +113,36 @@ function onStart(complete: () => void) {
             }),
             tg.wait(3),
 
-            // Part 2: Take outpost
-            // TODO: Camera pan on outpost
-            tg.immediate(_ => {
-                goalTakeOutpost.start();
+            tg.audioDialog(LocalizationKey.Script_4_Outpost_5, LocalizationKey.Script_4_Outpost_5, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
 
-                const dmgToDestroyTower = CreateDamageInfo(playerHero, playerHero, playerHero.GetAbsOrigin(), playerHero.GetAbsOrigin(), 9999, 9999);
+            tg.withHighlights(tg.seq([
+                tg.panCameraLinear(_ => playerHero.GetAbsOrigin(), outpostLocation, 2),
+                tg.audioDialog(LocalizationKey.Script_4_Outpost_6, LocalizationKey.Script_4_Outpost_6, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+                tg.audioDialog(LocalizationKey.Script_4_Outpost_7, LocalizationKey.Script_4_Outpost_7, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
 
-                const direTopTower1 = Entities.FindByName(undefined, "dota_badguys_tower1_top") as CDOTA_BaseNPC_Building | undefined;
-                if (direTopTower1 && unitIsValidAndAlive(direTopTower1)) {
-                    direTopTower1.TakeDamage(dmgToDestroyTower);
-                }
+                // Part 2: Take outpost
+                tg.immediate(_ => {
+                    goalTakeOutpost.start();
 
-                const direTopTower2 = Entities.FindByName(undefined, "dota_badguys_tower2_top") as CDOTA_BaseNPC_Building | undefined;
-                if (direTopTower2 && unitIsValidAndAlive(direTopTower2)) {
-                    direTopTower2.TakeDamage(dmgToDestroyTower);
-                }
-            }),
+                    const dmgToDestroyTower = CreateDamageInfo(playerHero, playerHero, playerHero.GetAbsOrigin(), playerHero.GetAbsOrigin(), 9999, 9999);
```

Comment:
I'm not sure if that's a foolproof method to use this. Is the second tower even necessary for this section? It might be better to simply remove it from the section altogether, similar to how most chapters don't begin with the top T1 tower after Chapter 2 Towers section.

We might want to simply kill the tower when the section begins if it's alive, this can be done with tower.Kill(undefined, undefined).

Reply:
I tried tower.Kill initially and outpost didn't get released for me to take somehow. Instead, TakeDamage works.